test/abilities/shields-down.test.ts

```diff
@@ -54,4 +56,130 @@ describe("Abilities - SHIELDS DOWN", () => {
 
     expect(minior.formIndex).toBe(meteorForm);
   });
+
+  test("should ignore non-volatile status moves", async () => {
+    game.override.enemyMoveset([MoveId.SPORE]);
+
+    await game.classicMode.startBattle([Species.MINIOR]);
+    game.move.use(MoveId.SPLASH);
+    await game.phaseInterceptor.to(TurnEndPhase);
+
+    expect(game.field.getPlayerPokemon()!.status).toBe(undefined);
```

Comment:
```suggestion
    expect(game.field.getPlayerPokemon().getStatusEffect()).toBe(StatusEffect.NONE);
```
Same goes for other instances of this line